TAF/testScenarios/functionalTest/V2-API/core-data/reading/GET-positive.robot

```diff
@@ -0,0 +1,63 @@
+*** Settings ***
+Resource         TAF/testCaseModules/keywords/commonKeywords.robot
+Suite Setup      Setup Suite
+
+*** Variables ***
+${SUITE}         Core-Data Reading GET Postive Testcases
+
+*** Test Cases ***
+ReadingGET001 - Query all readings
+    Given Create Multiple Events
+    When Query All Readings
+    Then Should Return Status Code "200"
+    And Should Have Content-Type "application/json"
+    And Readings Should Be Matched
+    [Teardown]  Delete Events
+
+ReadingGET002 - Query reading by ID
+    Given Create An Event
+    When Query Reading By ID
+    Then Should Return Status Code "200"
+    And Should Have Content-Type "application/json"
+    And Readings Should Be Matched
+    [Teardown]  Delete Events
+
+ReadingGET003 - Query all readings with specified device by device id
+    Given Create Multiple Events
+    When Query All Readings With Specified Device
+    Then Should Return Status Code "200"
+    And Should Have Content-Type "application/json"
+    And Readings Should Be Matched
+    [Teardown]  Delete Events
+
+ReadingGET004 - Query readings by start/end time
+    Given Create Multiple Events
+    When Query Readings By Start/End Time
+    Then Should Return Status Code "200"
+    And Should Have Content-Type "application/json"
+    And Readings Should Be Matched
+    [Teardown]  Delete Events
+
+ReadingGET005 - Query readings by valueType
+    Given Create Multiple Events
+    When Query Readings By valueType
+    Then Should Return Status Code "200"
+    And Should Have Content-Type "application/json"
+    And Readings Should Be Matched
+    [Teardown]  Delete Events
+
+ReadingGET005 - Query a count of all of readings
+    Given Create Multiple Events
+    When Query All Readings Count
+    Then Should Return Status Code "200"
+    And Should Have Content-Type "application/json"
+    And Count Should Be Correct
+    [Teardown]  Delete Events
+
+ReadingGET006 - Query a count of all of readings with specified device by device id
+    Given Create Multiple Events
```

Comment:
Create events on several devices and verify only return the count for the specified device.